FinSearchUnified/ShopwareProcess.php

```diff
@@ -224,6 +193,121 @@ public function getFindologicXml($lang = "de_DE", $start = 0, $length = 0, $save
         return $xmlDocument;
     }
 
+    /**
+     * @param int $productId
+     *
+     * @return null|string
+     */
+    public function getProductById($productId)
+    {
+        $xmlDocument = null;
+        $exporter = Exporter::create(Exporter::TYPE_XML);
+
+        $articlesQuery = $this->articleRepository->createQueryBuilder('articles')
+            ->select('articles')
+            ->where('articles.id = :productId')
```

Comment:
In my implementation I allowed everything that can be added to the `ordernumber` field. This makes the usage way more flexible.

See `ProductService::addProductIdFilters` in my implementation:
https://github.com/findologic/plugin-shopware-6/blob/72b892d7d1ec5802aa7db2353285986b1236c340/src/Export/ProductService.php#L109